presto-main/src/main/java/io/prestosql/operator/TaskContext.java

```diff
@@ -368,6 +373,14 @@ public Duration getFullGcTime()
         return new Duration(max(0, endFullGcTimeNanos - startFullGcTimeNanos), NANOSECONDS);
     }
 
+    /**
+     * verifies how many objects were moved to the Garbage Collector.
+     * <p>
+     * Verifies how many objects were started and how many objects were finished and returns
+     * the total that were moved to the Garbage Collector.
```

Comment:
"Check how many objects were instantiated and removed in the task context and then calculates the total quantity of objects that were collected by the Garbage Collector."